src/compute_pressure.cpp

```diff
@@ -140,6 +173,7 @@ void ComputePressure::init()
   nvirial = 0;
   vptr = NULL;
 
+  if (pairhybridflag && force->pair) nvirial++;
```

Comment:
There is one more thing that should/could be done: it should be checked in the `init()` function, if the pair style is still available. there could have been a pair style command changing it since the compute was defined. please see compute pair for comparison.

Reply:
Corrected. But I needed to reput `nsub` as in the class.